src/pages/home/report/ReportActionsView.js

```diff
@@ -145,6 +160,35 @@ class ReportActionsView extends React.Component {
         this.setState({refetchNeeded});
     }
 
+    /**
+     * Checks if the unreadActionIndicator should be shown.
+     * If it does, starts a timeout for the fading out animation and creates
+     * a flag to not show it again if the report is still open
+     */
+    setUpUnreadActionIndicator() {
+        if (!this.props.isActiveReport || !this.shouldShowNewActionIndicator) {
+            return;
+        }
+
+        this.unreadActionCount = _.find(this.props.reports, report => (
+            report.reportID === this.props.reportID
+        )).unreadActionCount;
+
+        if (this.unreadActionCount > 0) {
+            this.unreadIndicatorOpacity = new Animated.Value(1);
+            setTimeout(() => {
```

Comment:
I'm not sure if it matters that much, but it's a good practice to clean things like timers up when the component unmounts.

Reply:
I see. 
Something like:
1. Define a variable `this.unreadIndicatorTimer = null` on the constructor;
2. `this.unreadIndicatorTimer = setTimeout(() => ...)` in the `setUpUnreadActionIndicator` function;
3. `clearTimeout(this.unreadIndicatorTimer)` in `componentWillUnmount()`.

That's probably safer. I'll change it.
I'll also squash the commits, too many weird names.